### DISCUSSION TITLE
How to get the development environment to work?

### DISCUSSION THREAD

**Discussion by bebraw** [ssssota/svg2png-wasm#74] [Q&A] [answered]
I figured out I need pnpm and then `pnpm install` to get the development dependencies installed. `pnpm build` gives me this:

```bash
> svg2png-wasm@0.0.0-development build /Users/juhovepsalainen/Projects/github/svg2png-wasm
> run-s build:*


> svg2png-wasm@0.0.0-development build:wasm /Users/juhovepsalainen/Projects/github/svg2png-wasm
> wasm-pack build --target web --out-dir dist-wasm && npm run copy-wasm

[INFO]: 🎯  Checking for the Wasm target...
Error: wasm32-unknown-unknown target not found in sysroot: "/usr/local/Cellar/rust/1.66.0"

Used rustc from the following path: "/usr/local/bin/rustc"
It looks like Rustup is not being used. For non-Rustup setups, the wasm32-unknown-unknown target needs to be installed manually. See https://rustwasm.github.io/wasm-pack/book/prerequisites/non-rustup-setups.html on how to do this.
```

It feels like I'm missing something obvious or I skipped a step. Do you know how to get the Rust portion going?

**Comment by ssssota** [comment, marked as answer]
Thank you for your question.

You will need to install Rust for development.

Please follow steps 1 and 2 below: https://rustwasm.github.io/docs/wasm-pack/quickstart.html

**Reply by bebraw** [reply]
Ok, thanks. I have some issue compiling wasm-pack (https://github.com/rustwasm/wasm-pack/issues/1205) so I'll have another look after that has been resolved.
